doc/manual/manual.tex

```diff
@@ -7021,6 +7021,158 @@ \subsubsection{Using reconstructed surface velocities by GPlates}
+
+In this cookbook we will set up a compressible mantle convection model that uses 
+the (truncated) anelastic liquid approximation (see Sections~\ref{sec:ala} and 
+\ref{sec:tala}), together with a reference profile read in from an ASCII data file.
+The data we use here is generated with the open source mineral physics toolkit BurnMan 
+(\url{www.burnman.org}) using the python example program \texttt{simple\_adiabat.py}. 
+This file is available as a part of BurnMan, and provides a tutorial for how to generate 
+ASCII data files that can be used together with \aspect{}. 
+The computation is based on the Birch-Murnaghan equation of state, and uses a harzburgitic 
+composition. However, in principle, other compositions or equations of state can be used, 
+as long as the reference profile contains data for the reference temperature, pressure, density, 
+gravity, thermal expansivity, specific heat capacity and compressibility. Using BurnMan 
+to generate the reference profile has the advantage that all the material property data 
+are consistent, for example, the gravity profile is computed using the reference density.
+\begin{figure}
+  \includegraphics[width=\textwidth]{cookbooks/burnman/reference_profile.pdf}
+  \caption{\it Reference profile generated using BurnMan.}
+  \label{fig:burnman-reference-profile}
+\end{figure}
+The reference profile is shown in Figure~\ref{fig:burnman-reference-profile}, and the corresponding data file 
+is located at \url{data/adiabatic-conditions/ascii-data/isentrope_properties.txt}.
+
+\paragraph{Setting up the \aspect{} model.}
+In order to use this profile, we have to import and use the data in the adiabatic conditions 
+model, in the gravity model and in the material model, which is done using the corresponding 
+ASCII data plugins. The input file is provided in \url{cookbooks/burnman.prm}, and it uses the 
+2d shell geometry previously discussed in Section~\ref{sec:shell-simple-2d} and surface velocities
+imported from GPlates as explained in Section~\ref{sec:cookbooks-gplates}.
+
+To use the BurnMan data in the material model, we have to specify that we want to use the 
+\texttt{ascii reference profile} model. This material model makes use of the functionality provided by
+the \texttt{AsciiData} classes in \aspect{}, which allow plugins such as material models, 
+boundary or initial conditions models to read in ASCII data files (see for example Section~\ref{sec:geomio}). 
+Hence, we have to provide the directory and file name of the data to be used in the separate subsection 
+\texttt{Ascii data model}, and the same functionality and syntax will also be used for the 
+adiabatic conditions and gravity model.
+
+The viscosity in this model is computed as the product of a radial profile $\eta_r(z)$ and a 
+term that describes the dependence on temperature:
+\begin{align*}
+\eta(z,T) = \eta_r(z) \eta_0 \exp\left(-A \frac{T - T_\text{adi}}{T_\text{adi}}\right),
```

Comment:
Since we work in Cartesian coordinates, the use of `z` may be confusing without explanation.